tutorials/io/saving_games.rst

```diff
@@ -191,67 +155,76 @@ way to pull the data out of the file as well.
 
     # Note: This can be called from anywhere inside the tree. This function
     # is path independent.
     func load_game():
-        var save_game = File.new()
-        if not save_game.file_exists("user://savegame.save"):
+        var save_game = ConfigFile.new()
+        var err = save_game.load("user://savegame.save"):
+        if not err == OK:
             return # Error! We don't have a save to load.
 
         # We need to revert the game state so we're not cloning objects
         # during loading. This will vary wildly depending on the needs of a
         # project, so take care with this step.
         # For our example, we will accomplish this by deleting saveable objects.
-        var save_nodes = get_tree().get_nodes_in_group("Persist")
-        for i in save_nodes:
-            i.queue_free()
+        var save_nodes = get_tree().get_nodes_in_group("persist")
+        for node in save_nodes:
+            node.queue_free()
 
-        # Load the file line by line and process that dictionary to restore
-        # the object it represents.
-        save_game.open("user://savegame.save", File.READ)
-        while save_game.get_position() < save_game.get_len():
-            # Get the saved dictionary from the next line in the save file
-            var node_data = parse_json(save_game.get_line())
+        # Process each section in the file to restore the object it represents.
+        for section in save_game.get_sections():
+            var node_data = {}
+            # Adds each key in the section to the dictionary.
+            for key in save_game.get_section_keys(section):
+                node_data[key] = save_game.get_value(section, key)
 
             # Firstly, we need to create the object and add it to the tree and set its position.
             var new_object = load(node_data["filename"]).instance()
             get_node(node_data["parent"]).add_child(new_object)
-            new_object.position = Vector2(node_data["pos_x"], node_data["pos_y"])
+            new_object.position = node_data["pos"]
 
             # Now we set the remaining variables.
-            for i in node_data.keys():
-                if i == "filename" or i == "parent" or i == "pos_x" or i == "pos_y":
+            for key in node_data.keys():
+                if key == "filename" or key == "parent" or key == "pos":
                     continue
-                new_object.set(i, node_data[i])
+                new_object.set(key, node_data[key])
 
-        save_game.close()
 
  .. code-tab:: csharp
 
     // Note: This can be called from anywhere inside the tree. This function is
     // path independent.
     public void LoadGame()
     {
-        var saveGame = new File();
-        if (!saveGame.FileExists("user://savegame.save"))
-            return; // Error!  We don't have a save to load.
+        var saveGame = new ConfigFile();
+        var err =SaveGame.load("user://savegame.save") = saveGame.GD.Load("user://savegame.save");
```

Comment:
This looks weird loading twice. Copy/paste err?

Reply:
Yeah. ConfigFiles don't have to check if the file exists. I don't know C# but I'll try to fix it